app/entities/validator.go

```diff
@@ -0,0 +1,20 @@
+package entities
+
+import (
+	"regexp"
+
+	"github.com/go-devs-ua/octagon/app/globals"
+)
+
+const idMask = "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-4[a-fA-F0-9]{3}-[8|9|aA|bB][a-fA-F0-9]{3}-[a-fA-F0-9]{12}$" //nolint:gosec,lll // "Potential hardcoded credentials" regexp can`t be changed.
+
+var idRegex = regexp.MustCompile(idMask)
```

Comment:
Pls take a look at uuid.Parse() as an alternative for UUID validation.

Reply:
Just take a look? or have to use it?

Reply:
fixed